htslib/faidx.h

```diff
@@ -312,6 +312,8 @@ const char *fai_parse_region(const faidx_t *fai, const char *s,
 HTSLIB_EXPORT
 void fai_set_cache_size(faidx_t *fai, int cache_size);
 
+HTSLIB_EXPORT
+char *fai_path(const char *fa);
```

Comment:
Could you add some doxygen to say what this does, please?

Reply:
Does the caller have to `free()` the string returned?

Does the answer to that depend on whether the `fa` given had `##idx##` in it? (Hope not — that'd be tedious to use!)

Reply:
> Does the caller have to free() the string returned?

Yes, but in this respect, the method is not different in usage from the previous `samfaipath`. I'll add a note to the documentation to make it clearer.

> Does the answer to that depend on whether the fa given had ##idx## in it? (Hope not — that'd be tedious to use!)

No. In any scenario, the returned string is a dynamically allocated char array (or NULL).

Reply:
:+1: Sounds good, just needs mentioning in the doxygen doco.